critical/elastic_logging_cluster.tf

```
@@ -52,6 +52,14 @@ locals {
       }
     }
   }
+
+  logging_apm_user_settings = {
+    "apm-server" : {
+      rum : {
```

Just for the sake of anybody else who isn't super familiar with APM stuff, and thinks you're trying to get Elasticsearch drunk:

```suggestion
      # RUM = Real-User Monitoring
      rum : {
```